self_hosted/main.jou

```diff
@@ -90,6 +90,10 @@ def parse_args(argc: int, argv: byte**) -> CommandLineArgs:
 
     return result
 
+def find_file(files: FileState*, nfiles: int, path: byte*) -> FileState:
```

Comment:
You can return a pointer to an individual file, just like the C code. (See Jou tutorial or ask me to clarify if you are new to pointers.) By using pointers, you can use `NULL` as a "not found" value, and the compiler is happy because you always return something.

```suggestion
def find_file(files: FileState*, nfiles: int, path: byte*) -> FileState*:
```

Reply:
Thanks!